cmd/agent/main.go

```diff
@@ -552,7 +573,22 @@ func KubeletCertificateNeedsRegeneration(ipAddress, currentHostname string, cert
 		return false, err
 	}
 
-	return !CertificateIncludesHostname(currentHostname, parsedCert) || CertificateIsExpiring(parsedCert, currentTime) || !CertificateIncludesCurrentIP(ipAddress, parsedCert), nil
+	if !CertificateIncludesHostname(currentHostname, parsedCert) {
+		logrus.Tracef("Certificate does not include current hostname, requesting new certificate")
```

Comment:
nit 
```suggestion
		logrus.Tracef("certificate does not include current hostname, requesting new certificate")
```

Reply:
These are log messages and generally don't follow the requirement to start with lowercase letter as required by Go standards for _errors_.

Reply:
I've changed it to a lower case

Reply:
I see, other log messages introduced here start with lowercase character. I'm fine either way as soon as it's consistent.
Glancing through the code I see we're not consistent across the codebase with some a mix of messages starting with lowercase and uppercase.

Just wanted to make sure we're not misunderstanding https://github.com/golang/go/wiki/CodeReviewComments#error-strings which specifically states:
> This does not apply to logging, which is implicitly line-oriented and not combined inside other messages.